changelog.json

```
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "1.7.0-beta0": [
+      "[New] Add Recent Group to Repo List - #7132",
+      "[Added] Explicit refresh of PR list and avoid skeleton loading screen when we have PRs to show - #7027",
```

If you like this change in category, it'll need to be moved down to the bottom to maintain sort order: 

```suggestion
      "[Improved] Pull Request list now supports manual refresh to improve usability - #7027",
```